.pre-commit-config.yaml

```diff
@@ -18,9 +18,9 @@ repos:
       - id: codespell
         args:
           - --ignore-words-list=hass,alot,datas,dof,dur,farenheit,hist,iff,ines,ist,lightsensor,mut,nd,pres,referer,ser,serie,te,technik,ue,uint,visability,wan,wanna,withing
-          - --skip="./.*,*.json"
+          - --skip="./.*,*.csv,*.json"
           - --quiet-level=2
-        exclude_types: [json]
+        exclude_types: [csv, json]
```

Comment:
@frenck is this ok? We have a fixture csv file that we've included.

Reply:
Yeah that is fine 👍